install/autodetect.go

```diff
@@ -223,7 +232,7 @@ func (k *K8sInstaller) autodetectKubeProxy(ctx context.Context) error {
 			return nil
 		}
 	}
-	apiServerHost, apiServerPort := k.client.GetAPIServerHostAndPort()
```

Comment:
Hmm, why is it safe to remove this line? I agree that we must prefer user-provided inputs over autodetection, but why remove autodetection entirely?

Reply:
I've read through it a couple of more times and it's not safe. Also I don't think the flow of the autodetect stuff makes much sense with these changes. I will check tomorrow how it can be improved, regardless setting the helm values in the end of autodetect feels wrong.